openedx_learning/apps/authoring/backup_restore/toml.py

```diff
@@ -27,20 +29,25 @@ def toml_learning_package(learning_package: LearningPackage) -> str:
     return tomlkit.dumps(doc)
 
 
-def toml_publishable_entity(entity: PublishableEntityMixin) -> str:
+def toml_publishable_entity(entity: PublishableEntity) -> str:
     """Create a TOML representation of a publishable entity."""
+
+    current_draft_version = getattr(entity, "draft", None)
+    current_published_version = getattr(entity, "published", None)
+
     doc = tomlkit.document()
     entity_table = tomlkit.table()
     entity_table.add("uuid", str(entity.uuid))
     entity_table.add("can_stand_alone", entity.can_stand_alone)
 
-    draft = tomlkit.table()
-    draft.add("version_num", entity.versioning.draft.version_num)
-    entity_table.add("draft", draft)
+    if current_draft_version:
+        draft = tomlkit.table()
```

Comment:
Nit: It might be clearer to call this a `draft_table` so it's clear at a glance that this is not a `Draft` model object.